transformation_filter.proto

```diff
@@ -4,10 +4,11 @@ package envoy.api.v2.filter.http;
 
 option go_package = "transformation";
 
-// [#protodoc-title: Lambda]
-// Lambda :ref:`configuration overview <config_http_filters_lambda>`.
 message Transformations {
     map<string, Transformation> transformations = 1;
+    // route_specific_config and transformations should be in a oneof,
```

Comment:
add a comment that if `route_specific_config` is set, `transformations` here will be ignored (and that transformations instead will be looked up in the per-filter config on the route